t/v2/cluster.t

```diff
@@ -81,24 +81,28 @@ checked error msg as expect: Key not found
 all done
 
 
-
-=== TEST 2: cluster set + delete + get
+=== TEST 2: cluster set + delete + get + auth
```

Comment:
We'd better add a new test case for testing new case.

Reply:
cluster case is mian test auth